examples/conda_types.rs

```diff
@@ -0,0 +1,17 @@
+use rattler_conda_types::{PackageName, VersionSpec, Platform};
```

Comment:
I dont think this example is very useful. I think it makes much more sense to have these examples inside the rust code itself as part of the doc comments, they are then also verified.